src/app/common/modals/comments-modal/comments-modal.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="p-6">
+  @if (taskComment.commentType === 'image') {
+    <img class="w-full h-full p-0 rounded border-none" [src]="commentResourceUrl" />
+  } @else if (taskComment.commentType === 'pdf') {
+    <iframe
```

Comment:
Switch this to our PDF viewer... this will work on mobile devices now.

Reply:
replaced with f-pdf-viewer - before/after screenshots updated in the original comment